chapters/annotations.tex

```diff
@@ -1065,46 +1065,38 @@ \subsubsection{Text}\label{text}
   \%\% replaced by \%
 \item
-  \%name replaced by the name of the component (i.e., the identifier for
-  it in the enclosing class).
+  \%name replaced by the name of the component (i.e., the identifier for it in the enclosing class).
 \item
   \%class replaced by the name of the class (only the last part of the hierarchical name).
 \item
-  \%\emph{par} and \%\{\emph{par\}} replaced by the value of the
-  parameter \lstinline!par!.
-  If the value is numeric, tools shall display the value with \lstinline!displayUnit!, formatted according to bipm-specification.
+  \%\emph{par} and \%\{\emph{par}\} replaced by the value of the parameter \lstinline!par!.
+  If the value is numeric, tools shall display the value with \lstinline!displayUnit!, formatted according to the BIPM specification.
   E.g., for
 \begin{lstlisting}[language=modelica]
 parameter Real t(unit = "s", displayUnit = "ms") = 0.1
 \end{lstlisting}
   tools shall display \emph{100 ms}.
-  The intent is that the text is easily readable,
-  thus if \lstinline!par! is of an enumeration type, replace \lstinline!%par! by the item name,
-  not by the full name.
+  The intent is that the text is easily readable, thus if \lstinline!par! is of an enumeration type, replace \%\emph{par} by the item name, not by the full name.
   \begin{example}
-  If \lstinline!par = "Modelica.Blocks.Types.Enumeration.Periodic"!, then \lstinline!%par! should be displayed as \emph{Periodic}.
+  If \lstinline!par = "Modelica.Blocks.Types.Enumeration.Periodic"!, then \%\emph{par} should be displayed as \emph{Periodic}.
   \end{example}
-  The form \%\{\emph{par\}} allows component-references and is required for quoted identifiers, and can be directly
-  followed by a letter. Thus \lstinline!%{w}x%{h}! gives the value of \lstinline!w!
-  directly followed by \emph{x} and the value of \lstinline!h!, while \lstinline!%wxh! gives the value of the
-  parameter \lstinline!wxh!. If the parameter does not exist it is an error.
+  When quoted identifiers (e.g., \lstinline!rec.'}'.'quoted ident'!) are involved, the form \%\{\emph{par}\} must be used.
+  Here, \emph{par} is a general \lstinline[language=grammar]!component-reference!, and the macro can be directly followed by a letter.
+  Thus \lstinline!%{w}x%{h}! gives the value of \lstinline!w! directly followed by \emph{x} and the value of \lstinline!h!, while \lstinline!%wxh! gives the value of the parameter \lstinline!wxh!.
+  If the parameter does not exist it is an error.
 \end{itemize}
 
-The style attribute \lstinline!fontSize! specifies the font size. If the \lstinline!fontSize!
-attribute is 0 the text is scaled to fit its extent. Otherwise, the size
-specifies the absolute size. The text is vertically centered in the extent.
+The style attribute \lstinline!fontSize! specifies the font size. If the \lstinline!fontSize! attribute is 0 the text is scaled to fit its extent. Otherwise, the size specifies the absolute size. The text is vertically centered in the extent.
```

Comment:
```suggestion
The style attribute \lstinline!fontSize! specifies the font size.
If the \lstinline!fontSize! attribute is 0 the text is scaled to fit its extent.
Otherwise, the size specifies the absolute size.
The text is vertically centered in the extent.
```
If we are using sentence-lines we should do it consistently.